bpf/lib/proxy.h

```diff
@@ -346,16 +339,32 @@ ctx_redirect_to_proxy_first(struct __ctx_buff *ctx, __be16 proxy_port)
 	}
 #endif /* ENABLE_IPV4 */
 	default:
```

Comment:
is the `default` statement actually required - or could it be removed completely? (best practice, convention, compile issues if ipv4&ipv6 aren't enabled (empty switch statement), ...)